pkg/manager/impl/execution_manager.go

```diff
@@ -1111,15 +1111,16 @@ func (m *ExecutionManager) GetExecutionData(
 		Inputs:  &inputsURLBlob,
 	}
 	maxDataSize := m.config.ApplicationConfiguration().GetRemoteDataConfig().MaxSizeInBytes
-	if maxDataSize == 0 || inputsURLBlob.Bytes < maxDataSize {
+	remoteDataScheme := m.config.ApplicationConfiguration().GetRemoteDataConfig().Scheme
+	if remoteDataScheme == common.Local || inputsURLBlob.Bytes < maxDataSize {
```

Comment:
what if we change it to always return inline and sometimes return the signed ulr?

Reply:
then we should just get rid of the signed url altogether, no?

Reply:
Thats what I want to do, but I dont know if that will work for all cases at Lyft? If we have a metric lets collect that?

Reply:
we have a metric for observing the bytes size - can follow up to find out what it trends towards